docs2/site/docs/getting-started/errors.md

```diff
@@ -1,78 +1,149 @@
 # Error Handling
 
-The `ExecutionResult` provides an `Errors` property which includes any errors encountered
-during execution. Errors are returned [according to the spec](https://graphql.github.io/graphql-spec/June2018/#sec-Errors),
-which means stack traces are excluded. The `ExecutionResult` is transformed to what the spec
-requires using one or the another `IDocumentWriter`.
+Errors within the GraphQL engine can be thought of as falling into one of three groups: schema errors, input errors, and processing errors.  For instance, if an invalid query was passed to the `DocumentExecuter`, it would be considered an input error, and a `SyntaxError` would be thrown. Or if an invalid enum string was passed as a variable to a query, an `InvalidValueError` would be thrown. All validation rules that fail their respective tests are treated as input errors.  Processing errors typically represent unanticipated exceptions occurring during the execution of a field resolver, such as a timeout during a database operation.
```

Comment:
```suggestion
Errors within the GraphQL engine can be thought of as falling into one of three groups: schema errors, input errors and processing errors. For instance, if an invalid query was passed to the `DocumentExecuter`, it would be considered an input error, and a `SyntaxError` would be thrown. Or if an invalid enum string was passed as a variable to a query, an `InvalidValueError` would be thrown. All validation rules that fail their respective tests are treated as input errors.  Processing errors typically represent unanticipated exceptions occurring during the execution of a field resolver, such as a timeout during a database operation.
```